app/src/main/res/layout/fragment_profile.xml

```diff
@@ -2,124 +2,262 @@
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/transparent" />
+
+            <de.hdodenhof.circleimageview.CircleImageView
+                android:id="@+id/profile_picture_image_view"
+                android:layout_width="@dimen/profile_screen_circle_image_view"
+                android:layout_height="@dimen/profile_screen_circle_image_view"
+                android:layout_marginTop="@dimen/profile_screen_margin_top_circle_image_view"
+                android:layout_marginBottom="@dimen/profile_screen_margin_bottom_circle_image_view"
+                app:civ_border_color="@color/white"
+                app:civ_border_width="@dimen/profile_screen_border_circle_image_view"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <TextView
+                android:id="@+id/name_text_view"
+                style="@style/TextAppearance.AppCompat.Widget.ActionBar.Title"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:fontFamily="@font/roboto_medium"
+                android:letterSpacing="0.15"
+                android:lineHeight="@dimen/profile_screen_name_line_height_size"
+                android:textColor="@android:color/white"
+                android:textSize="@dimen/profile_screen_name_font_size_20sp"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/profile_picture_image_view" />
+
+            <TextView
+                android:id="@+id/grade_text_view"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:fontFamily="@font/roboto_regular"
+                android:letterSpacing="0.4"
+                android:lineHeight="@dimen/profile_screen_line_height_size_16sp"
+                android:textColor="@android:color/white"
+                android:textSize="@dimen/profile_screen_grade_font_size_12sp"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/name_text_view" />
+
+
+            <androidx.recyclerview.widget.RecyclerView
+                android:id="@+id/tags_recycler_view"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/profile_screen_recycle_view_margin_top"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/grade_text_view"
+                />
 
             <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
There is too much space at the end of this view. Use the same distance between 'Other Information' view and navBar that you used between all other views